mne/channels/montage.py

```diff
@@ -131,50 +130,51 @@ def read_montage(kind, ch_names=None, path=None, unit='m', transform=False):
     standard_1005         Electrodes are named and positioned according to the
-                          international 10-05 system.
+                          international 10-05 system (346 channels)
     standard_1020         Electrodes are named and positioned according to the
-                          international 10-20 system.
+                          international 10-20 system (97 channels)
     standard_alphabetic   Electrodes are named with LETTER-NUMBER combinations
-                          (A1, B2, F4, etc.)
+                          (A1, B2, F4, ...) (68 channels)
     standard_postfixed    Electrodes are named according to the international
                           10-20 system using postfixes for intermediate
-                          positions.
+                          positions (103 channels)
     standard_prefixed     Electrodes are named according to the international
                           10-20 system using prefixes for intermediate
-                          positions.
+                          positions (77 channels)
     standard_primed       Electrodes are named according to the international
                           10-20 system using prime marks (' and '') for
-                          intermediate positions.
-
-    biosemi16             BioSemi cap with 16 electrodes
-    biosemi32             BioSemi cap with 32 electrodes
-    biosemi64             BioSemi cap with 64 electrodes
-    biosemi128            BioSemi cap with 128 electrodes
-    biosemi160            BioSemi cap with 160 electrodes
-    biosemi256            BioSemi cap with 256 electrodes
-
-    easycap-M10           Brainproducts EasyCap with electrodes named
-                          according to the 10-05 system
-    easycap-M1            Brainproduct EasyCap with numbered electrodes
-
-    EGI_256               Geodesic Sensor Net with 256 channels
-
-    GSN-HydroCel-32       HydroCel Geodesic Sensor Net with 32 electrodes
-    GSN-HydroCel-64_1.0   HydroCel Geodesic Sensor Net with 64 electrodes
-    GSN-HydroCel-65_1.0   HydroCel Geodesic Sensor Net with 64 electrodes + Cz
-    GSN-HydroCel-128      HydroCel Geodesic Sensor Net with 128 electrodes
-    GSN-HydroCel-129      HydroCel Geodesic Sensor Net with 128 electrodes + Cz
-    GSN-HydroCel-256      HydroCel Geodesic Sensor Net with 256 electrodes
-    GSN-HydroCel-257      HydroCel Geodesic Sensor Net with 256 electrodes + Cz
+                          intermediate positions (103 channels)
+
+    biosemi16             BioSemi cap with 16 electrodes (19 channels)
+    biosemi32             BioSemi cap with 32 electrodes (35 channels)
+    biosemi64             BioSemi cap with 64 electrodes (67 channels)
+    biosemi128            BioSemi cap with 128 electrodes (131 channels)
+    biosemi160            BioSemi cap with 160 electrodes (163 channels)
+    biosemi256            BioSemi cap with 256 electrodes (259 channels)
```

Comment:
The `standard` montages seem to have more fiducials:  Nz, LPA, RPA, A1, A2, M1, M2. That's what I meant, it is difficult to distinguish between real channel labels and fiducials, so I'm not sure if it's worth the effort. IMO it's sufficient to state this generally before, as it is currently implemented.